### DISCUSSION TITLE
Does NeMo data manifest format support long audio?

### DISCUSSION THREAD

**Discussion by feddybear** [NVIDIA-NeMo/NeMo#2472] [Q&A] [answered]
As the title suggests, does the data manifest only assume that each line starts from the beginning of the file referred to by audio_filepath?
Is it not possible to have a long recording where you know the utterance boundaries and use the same file, indicate both start time and duration?

Sorry if I haven't spent much time finding this information from all documentations, just starting to ask from going over the tutorial

**Comment by titu1994** [comment, marked as answer]
Yes it supports it. You can use "offset" and "duration" keys in seconds in order to select a subset of the audio file.

**Reply by titu1994** [reply]
However you will need multiple rows for the same audio file to select different subsegments

**Reply by feddybear** [reply]
Yes, no problem with that. I'm just glad I don't have to trim them.  Thanks!!!
